test/unittests/utils_test.cpp

```diff
@@ -5,10 +5,10 @@
 #include <gtest/gtest.h>
 #include <test/utils/utils.hpp>
 
-TEST(utils, to_hex)
+TEST(utils, hex)
 {
     auto data = bytes{0x0, 0x1, 0xa, 0xf, 0x1f, 0xa0, 0xff, 0xf0};
```

Comment:
There were other places where you added a `const` modifier to not-modifying variables. Maybe here, too? (Depending on your general opinion about constness)

Reply:
I usually add `const` but these are just old unit tests, so it is probably not worth doing.